pkgs/development/lua-modules/overrides.nix

```diff
@@ -332,15 +332,23 @@ with prev;
   # we shouldn't use luarocks machinery to build complex cmake components
   libluv = pkgs.stdenv.mkDerivation {
 
-    inherit (prev.luv) pname version meta src;
+    pname = "libluv";
+    inherit (prev.luv) version meta src;
 
       cmakeFlags = [
         "-DBUILD_SHARED_LIBS=ON"
         "-DBUILD_MODULE=OFF"
         "-DWITH_SHARED_LIBUV=ON"
+        "-DLUA_BUILD_TYPE=System"
+        "-DWITH_LUA_ENGINE=${if isLuaJIT then "LuaJit" else "Lua"}"
       ];
 
-      buildInputs = [ pkgs.libuv ];
+      # to make sure we dont use bundled deps
+      postUnpack = ''
+        rm -rf deps/lua deps/libuv
+      '';
+
+      buildInputs = [ pkgs.libuv final.lua ];
```

Comment:
Why is `final.lua` all of sudden needed?

Reply:
I get a
```
       > -- Could NOT find Lua (missing: LUA_LIBRARIES LUA_INCLUDE_DIR)
       > CMake Error at CMakeLists.txt:146 (message):
       >   Failed to find Lua headers.  Variable `LUA_INCLUDE_DIR' expected to be
       >   defined.
```
without it. I think it used to work because it was using the bundled lua or they changed something in their build: for instance I had to add    "-DLUA_BUILD_TYPE=System"  that didn't seem to exist before